ocaml/typing/ctype.ml

```diff
@@ -2677,6 +2677,11 @@ let rec expands_to_datatype env ty =
       end
   | _ -> false
 
+let equivalent_in_classic_mode l1 l2 =
```

Comment:
I know the check is for `Clflags.classic`, but I find that name very strange. Let's rename this to `equivalent_with_nolabels`.